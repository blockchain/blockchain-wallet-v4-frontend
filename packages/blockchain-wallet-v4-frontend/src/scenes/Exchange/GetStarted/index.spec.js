import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { GetStarted } from './index'

jest.mock('blockchain-info-components', () => ({
  Image: 'image',
  Text: 'text'
}))
jest.mock('./StatusBar', () => 'StatusBar')

describe('StatusBar', () => {
  it('renders correctly', () => {
    const component = shallow(<GetStarted />)
    const tree = toJson(component)
    expect(tree).toMatchSnapshot()
  })
})
