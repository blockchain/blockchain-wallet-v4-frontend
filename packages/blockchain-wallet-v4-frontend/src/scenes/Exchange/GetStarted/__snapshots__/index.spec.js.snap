// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar renders correctly 1`] = `
<GetStarted__Wrapper>
  <GetStarted__Container>
    <GetStarted__Column>
      <GetStarted__Row
        marginBottom="36px"
      >
        <text
          color="brand-primary"
          size="30px"
          weight={600}
        >
          <FormattedMessage
            defaultMessage="Trading your crypto doesn't mean trading away control"
            id="scenes.exchange.getstarted.header"
            values={Object {}}
          />
        </text>
      </GetStarted__Row>
      <GetStarted__Row
        marginBottom="24px"
      >
        <GetStarted__DarkText>
          <FormattedMessage
            defaultMessage="Swap enables you to trade crypto with the best prices and quick settlement, all while maintaining full control of your funds."
            id="scenes.exchange.getstarted.description"
            values={Object {}}
          />
        </GetStarted__DarkText>
      </GetStarted__Row>
      <GetStarted__Row
        marginBottom="8px"
      >
        <GetStarted__DarkText>
          <GetStarted__PrimaryText>
            <FormattedMessage
              defaultMessage="Low Fees"
              id="scenes.exchange.getstarted.low_fees"
              values={Object {}}
            />
          </GetStarted__PrimaryText>
          <GetStarted__DarkText
            hideOnMobile={true}
          >
             - 
          </GetStarted__DarkText>
          <FormattedMessage
            defaultMessage="Great news! Get the best prices without having to leave the security of your Wallet."
            id="scenes.exchange.getstarted.low_fees_description"
            values={Object {}}
          />
        </GetStarted__DarkText>
      </GetStarted__Row>
      <GetStarted__Row
        marginBottom="8px"
      >
        <GetStarted__DarkText>
          <GetStarted__PrimaryText>
            <FormattedMessage
              defaultMessage="Higher Limits"
              id="scenes.exchange.getstarted.higher_limits"
              values={Object {}}
            />
          </GetStarted__PrimaryText>
          <GetStarted__DarkText
            hideOnMobile={true}
          >
             - 
          </GetStarted__DarkText>
          <FormattedMessage
            defaultMessage="Like to trade big? Get access to limits of up to $25,000 per day."
            id="scenes.exchange.getstarted.higher_lits_description"
            values={Object {}}
          />
        </GetStarted__DarkText>
      </GetStarted__Row>
      <GetStarted__Row
        marginBottom="32px"
      >
        <GetStarted__DarkText>
          <GetStarted__PrimaryText>
            <FormattedMessage
              defaultMessage="Easy to use"
              id="scenes.exchange.getstarted.easy_to_use"
              values={Object {}}
            />
          </GetStarted__PrimaryText>
          <GetStarted__DarkText
            hideOnMobile={true}
          >
             - 
          </GetStarted__DarkText>
          <FormattedMessage
            defaultMessage="Get started in just a few steps."
            id="scenes.exchange.getstarted.easy_to_use_description"
            values={Object {}}
          />
        </GetStarted__DarkText>
      </GetStarted__Row>
      <GetStarted__Row>
        <StatusBar />
      </GetStarted__Row>
    </GetStarted__Column>
    <GetStarted__Column>
      <GetStarted__PreviewImage />
    </GetStarted__Column>
  </GetStarted__Container>
</GetStarted__Wrapper>
`;
